ui/shared/status/StatusButton.qml

```diff
@@ -68,8 +68,7 @@ Button {
             anchors.right: iconLoader.active ? undefined : parent.right
             anchors.left: iconLoader.active ? iconLoader.right : parent.left
             anchors.leftMargin: iconLoader.active ? Style.current.smallPadding : 0
-            color: !enabled ? Style.current.buttonDisabledForegroundColor : 
-              (hovered || highlighted) ? Style.current.blue : control.color
+            color: !enabled ? Style.current.buttonDisabledForegroundColor : control.color
```

Comment:
I think you still need this part ` (hovered || highlighted) ? Style.current.blue : control.color`

You probably need to add a check for `warn` too

```
 (type !== "warn" && (hovered || highlighted)) ? Style.current.blue : control.color
 ```

Reply:
Currently, it is working well, however, let me follow you ;)